tensorflow-cpu.yaml

```diff
@@ -0,0 +1,126 @@
+  pypi-package: tensorflow-cpu
+  prefix: /usr
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - py${{vars.py-version}}-pip
+      - python-${{vars.py-version}}
+  environment:
+    # This is needed to work around the error "ValueError: ZIP does not support timestamps before 1980"
+    SOURCE_DATE_EPOCH: 315532800
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 35a7dbc754193523e5582ce5b5cbbb4c07cff7b2
+      repository: https://github.com/tensorflow/build
+      branch: master
+
+  - working-directory: tensorflow_runtime_dockerfiles
+    pipeline:
+      - name: Install cpu requirements
+        runs: |
+          pip${{vars.py-version}} install \
+            --no-cache-dir \
+            --root ${{targets.contextdir}} \
+            --prefix ${{vars.prefix}} \
+            -r cpu.requirements.txt
+      - name: Install Tensorflow
+        runs: |
+          pip${{vars.py-version}} install \
+            --no-cache-dir \
+            --root ${{targets.contextdir}} \
+            --prefix ${{vars.prefix}} \
+            ${{vars.pypi-package}}
+      - name: Install jupyter requirements
+        runs: |
+          pip${{vars.py-version}} install \
+            --no-cache-dir \
+            --root ${{targets.contextdir}} \
+            --prefix ${{vars.prefix}} \
+            -r jupyter.requirements.txt \
+            -U
+      - name: Install bashrc
+        runs: |
+          mkdir -p ${{targets.contextdir}}/etc
+          cp bashrc ${{targets.contextdir}}/etc/bash.bashrc
+      - name: Install tensorflow tutorials
+        runs: |
+          mkdir -p ${{targets.contextdir}}/tf/tensorflow-tutorials
+          cp jupyter.readme.md ${{targets.contextdir}}/tf/tensorflow-tutorials/README.md
+      - name: Cleanup pycache
+        runs: |
+          find . -name '__pycache__' -exec rm -rf {} +
+
+test:
+  environment:
+    contents:
+      packages:
+        - curl
+  pipeline:
+    - name: Test import of IPython kernel for Jupyter
+      uses: python/import
+      with:
+        imports: |
+          import ipykernel.kernelspec
+    - name: Test Tensorflow library
+      runs: |
+        cat <<"EOF" >test.py
+        import tensorflow as tf
+
+        # Check TensorFlow version
+        print("TensorFlow version:", tf.__version__)
+
+        # Check if TensorFlow can access the CPU
+        print("Is built with CUDA:", tf.test.is_built_with_cuda())
+        print("Available devices:")
+        for device in tf.config.list_physical_devices():
+            print(device)
+
+        # Perform a simple computation
+        a = tf.constant([1.0, 2.0, 3.0])
+        b = tf.constant([4.0, 5.0, 6.0])
+        c = tf.add(a, b)
+        print("Result of addition:", c.numpy())
+        EOF
+        python${{vars.py-version}} test.py
+    - name: Test Jupyter notebook run
+      runs: |
+        jupyter notebook --notebook-dir=/tf --ip 0.0.0.0 --no-browser --allow-root >/dev/null 2>&1 &
```

Comment:
We have a pipeline for daemons we could reuse here, something like `test/daemon-check-output`